src/home/sorting/sorting-manager.ts

```diff
@@ -9,6 +9,8 @@ export class SortingManager {
   private _filterTextBox: HTMLInputElement;
   private _sortingButtons: HTMLElement;
   private _instanceId: string;
+  private _sortingState: { [key: string]: "unsorted" | "ascending" | "descending" } = {}; // Track state for each sorting option
+  private _filterTimeout: number | null = null; // Timeout ID for debouncing
```

Comment:
I think we should remove the debounce logic.

Reply:
yes, ill remove it